libs/sdk-ui-dashboard/src/dashboard/commandHandlers/rootCommandHandler.ts

```diff
@@ -0,0 +1,35 @@
+// (C) 2021 GoodData Corporation
+import { actionChannel, call, getContext, take } from "redux-saga/effects";
+import { DashboardCommands } from "../../commands/dashboard";
+import { loadDashboardCommandHandler } from "./loadDashboardCommandHandler";
+import { DashboardContext } from "../state/dashboardStore";
+
+/**
+ * Root command handler is the central point through which all command processing is done. The handler registers
+ * for all actions starting with `GDC.CMD` === all dashboard commands.
```

Comment:
I wonder if this convention is scalable. What about doing GDC.DASHBOARD.CMD.*? In other words, should we namespace the names in case there will be other such system for idk Insights or others?

Reply:
fixed this and the events as well.